app/models/manageiq/providers/openstack/cloud_manager/vm/associate_ip.rb

```diff
@@ -0,0 +1,31 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::AssociateIp
+  def validate_associate_floating_ip
+    {:available => true, :message => nil}
+  end
+
+  def validate_disassociate_floating_ip
+    {:available => true, :message => nil}
```

Comment:
Maybe make disassociate action available only if there is at least one FloatingIP associated with the Vm.